docs/config.md

```diff
@@ -197,6 +201,47 @@ names:
   commonNamePattern: "{host}.buoyant.io"
 ```
 
+#### Load Balancer
+
```

Comment:
also, per @stevej, i think we'd _like_ to change the default for routers to be EWMA...